presto-native-execution/scripts/setup-centos.sh

```diff
@@ -54,6 +54,12 @@ function install_proxygen {
   cmake_install -DBUILD_TESTS=OFF -DBUILD_SHARED_LIBS=ON
 }
 
+function install_gh {
```

Comment:
Is this only required for github actions? Can we install in the github workflow?

Reply:
It is only used for the github workflow, yes. I originally had it as a step. I can put it back.

It is needed for the caching of ccache.